### ISSUE TITLE
Nicehash not recognizing my card

### ISSUE THREAD

**Issue by Roomerkind** [nicehash/NiceHashMiner#743]
[log.txt](https://github.com/nicehash/NiceHashMinerLegacy/files/1578315/log.txt)
After I updated my AMD card 7800 series to the latest driver 17.12.1 the Nicehash program does not recognize my card anymore. It recognized it fine for weeks and even tonight before I updated my drivers, now it does not.

**Comment by kochetkovd** [comment]
Got same problem. On 17.11.4 its okay but on 17.12.1/2 its didnt recognize the card.
AMD RX 560

**Comment by GrimHedgehog** [comment]
Same Problem. I am using some AMD 460's, updated to 17.12.1 and nhml won't recognize it.

**Comment by ghost** [comment]
same with rx580, 17.12.1

**Comment by AlienUrineSampl** [comment]
Revert driver back to 17.11 series (aka Crimson) NHML will not recognize 17.12(Adrenaline) drivers

**Comment by Roomerkind** [comment]
Hopefully NHML will be upgraded to work with the latest drivers! Yep I reverted and it recognizes my old 7800. I just bought a RX 570 only to get it home to find out that it needs an 8 pin power connector while my old card has a 6 pin. Drove back to the computer store to find out they don't have the needed adaptor in stock- I have two six pin connectors that I can use an adaptor on for my new RX 570. So I ordered the adaptor on Amazon. This has nothing to do with the NHML, I just want to vent!
